### PR TITLE
icon simplification

### PR DESCRIPTION
**Pull request by jmzwar**
Remove duplicated props and simplify usage.

### CODE REVIEW

File: v2/components/icons/ChevronDown/ChevronDown.tsx [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import React from 'react';
```
[the remark this answers is not in the document]
Comment (jmzwar) [reply v2/components/icons/ChevronDown/ChevronDown.tsx:1]: Right - I think we may need to update a typescript config then. Do you get a ts error on master?

[the remark this answers is not in the document]
Comment (bachstatter) [reply v2/components/icons/ChevronDown/ChevronDown.tsx:1]: What's your opinion about this, I sorta like having React imported in tsx files but that is probably just habit.

[the remark this answers is not in the document]
Comment (jmzwar) [reply v2/components/icons/ChevronDown/ChevronDown.tsx:1]: I've gotten accustomed to not using it. It does get a bit annoying having to add to every file.

**Review by 0xjocke — APPROVED** [review]
Nice, another thing that I try to do with icons befopre turning them into react components is to compress them with https://jakearchibald.github.io/svgomg/.
Not sure if you been doing that @jmzwar but its an easy win

**Comment by jmzwar** [comment]
@bachstatter It's less of a problem I find with figma. Definitely with sketch the entire kitchen sink was exported. I'll keep it in mind if they're getting big.

**Review by 0xjocke — APPROVED** [review]

**Review by noisekit — APPROVED** [review]

**Pull request merged**